internal/mount/mount.go

```diff
@@ -14,37 +14,33 @@ import (
 	"strings"
 )
 
-const (
-	csiDevicePath       = "/dev/csi0"
-	stateDiskMountPoint = "/mnt/state"
-)
-
-// setupMount mounts the csi device to the state disk mount point.
-func setupMount(ctx context.Context, logger *slog.Logger) error {
-	blk, err := blkid(ctx, csiDevicePath)
+// SetupMount formats the csi device to ext4 and mounts it to the provided mount point.
```

Comment:
The docstring could mention that the disk is only formatted if it's not already formatted.